File: C++/D201.cpp
#include <iostream>
using namespace std;

int main() {
    int Times;
    cin >> Times;
    if (Times == 0) {
        cout << 0 << endl;
    }
    else if (Times == 1) {
        cout << 1 << endl;
    }
    else {
        Times -= 1;
        int OldNumber;
        int NewNumber;
        int PrintNumber;
        OldNumber = 0;
        NewNumber = 1;
        for (int i = 0;i < Times;++i) {
            PrintNumber = OldNumber + NewNumber;
            OldNumber = NewNumber;
            NewNumber = PrintNumber;
        }
        cout << PrintNumber << endl;
    }
    return 0;
}

File: Python/D201.py
Times = int(input())
if Times == 0:
  print(0)
elif Times == 1:
  print(1)
else:
  Times = Times - 1
  OldNumber = 0
  NewNumber = 1
  for i in range(Times):
  	PrintNumber = OldNumber + NewNumber
  	OldNumber = NewNumber
  	NewNumber = PrintNumber
  print(PrintNumber)
